pom.xml

```diff
@@ -20,6 +20,7 @@
 
   <properties>
     <java.level>8</java.level>
+    <useBeta>true</useBeta>
```

Comment:
Seems like LogStorage is currently still in Beta, which is why this is necessary.